assets/js/vueHeroComponent.js

```diff
@@ -0,0 +1,44 @@
+Vue.component('hero', {
```

Comment:
If you're willing to set up Webpack, you can use single-file components in Vue. They will make writing Vue much more enjoyable!

https://vuejs.org/v2/guide/single-file-components.html